### DISCUSSION TITLE
增加predictions文件中保存的字段

### DISCUSSION THREAD

**Discussion by PamKing7** [open-compass/opencompass#425] [Q&A]
现在只保存了prompt与prediction，我想保存correct_answer字段，用于后续的分析，请问该如何修改代码

**Comment by gaotongxiao** [comment]
It's already implemented in https://github.com/open-compass/opencompass/pull/419 and will be included in the upcoming release, you may apply this PR locally if needed.

**Reply by PamKing7** [reply]
Can you give me a sample of the format of the saved JSON file？
